ballet/eng/misc.py

```diff
@@ -13,6 +15,24 @@ def __init__(self):
         super().__init__(funcy.identity)
 
 
+class BoxCoxTransformer(BaseTransformer):
+    def __init__(self, threshold, lmbda=0):
+        super().__init__()
+        self.threshold = threshold
+        self.lmbda = lmbda
+
+    def fit(self, X, **fit_args):
+        X = X.copy()
+        skewed_feats = X.apply(lambda x: skew(x.dropna()))
+        feats_to_transform = skewed_feats[abs(skewed_feats) > self.threshold]
+        for feat in feats_to_transform.index:
+            X[feat] = boxcox1p(X[feat], self.lmbda)
```

Comment:
It looks like the transformation is applied in the "fit" stage. The idea here is to determine which columns are skewed enough that they merit boxcox transformation. So in the `fit`, we just compute the skewed columns. Later, in the `transform`, we apply the transformation.

```
self.features_to_transform_ = features_to_transform
```
The trailing underscore is convention in sklearn to differentiate parameters that are passed into the constructor vs those that are fit during a training stage.

Alternately, can name this attribute `indices_`, `selected_`, `skewed_features_`, `skew_indices_` or so on